mkdocs_monorepo_plugin/plugin.py

```diff
@@ -65,6 +66,8 @@ def on_pre_page(self, page, config, files):
         if len(self.files_source_dir) > 0:
             if page.file.abs_src_path in self.files_source_dir:
                 page.file.abs_src_path = self.files_source_dir[page.file.abs_src_path]
+                if not is_index(page):
```

Comment:
Looks like `is_index` is only used within `set_edit_url`.  Would it make sense to only import `set_edit_url` and keep the `is_index` logic within that function?